cltk/wrappers/stanford.py

```diff
@@ -0,0 +1,206 @@
+"""Wrapper for the Python Stanford NLP package `stanfordnlp`.
+More here: <https://github.com/stanfordnlp/stanfordnlp>.
+"""
+
+import os
+
+import stanfordnlp  # type: ignore
+from typing import Dict
+from typing import Optional
+
+from cltk.exceptions import UnknownLanguageError
+from cltk.utils import file_exists
+
+
+class StanfordNLPWrapper:
+    """CLTK's wrapper for the `stanfordnlp` project."""
+
+    def __init__(self, language: str, treebank: Optional[str] = None) -> None:
```

Comment:
@free-variation this class so far just does some basic setup -- seems to me that since we're depending on a relatively new project, it'd make sense to do some extra setup checks (thinks like language name mapping, checks for defaults, model present, etc).

I haven't finished it, but my immediate goal is to have an example like (this would normally be called inside the class `cltk.nlp.NLP()`):

``` python
>>> from cltk.wrappers.stanford import StanfordNLPWrapper
>>> stanford_wrapper = StanfordNLPWrapper(language='latin')
>>> stanford_wrapper.parse('Arma virumque cano')
```

Then `cltk.NLP()` class would have some attribute like `.stanford_nlp_obj`, which we (to start) would leave for the user to parse. From here, we can explore how/whether to bind particular parts of the `stanford_nlp_obj` attribute to the `cltk.NLP` class.

I think of this approach to "building up" our wrapper, since we know we will want to return the entire object but leaves open smarter/finer parsing, too (as I understood your last comment in #5).

re #5